pkg/controller/build/buildrequest/buildrequestopts.go

```diff
@@ -196,13 +202,20 @@ func (o *optsGetter) getOpts(ctx context.Context, mosb *mcfgv1alpha1.MachineOSBu
 		return nil, fmt.Errorf("could not retrieve machineconfig %s: %w", mosb.Spec.DesiredConfig.Name, err)
 	}
 
+	cc, err := o.mcfgclient.MachineconfigurationV1().ControllerConfigs().Get(ctx, ctrlcommon.ControllerConfigName, metav1.GetOptions{})
```

Comment:
praise: Nice! This will make getting the global pull secret much easier when we make our UX improvements in 4.19.